extension/package.json

```diff
@@ -554,6 +554,12 @@
           "type": "string",
           "default": null
         },
+        "dvc.expTableHeadMaxLayers": {
```

Comment:
If you update the config, you need to restart the window to see the change to the table. Is it supposed to change once you update the config without restarting?

Reply:
Table updates on config change now!